src/m365/commands/login.ts

```diff
@@ -151,6 +197,88 @@ class LoginCommand extends Command {
     await this.initAction(args, logger);
     await this.commandAction(logger, args);
   }
+
+  private tokensForMultipleTenants(accessTokenValue: string | string[] | undefined, tenantValue: string | undefined): boolean {
```

Comment:
Let's rename this command to make it clearer what it does. Right now, without looking at its implementation, it's hard to tell what it's doing

Reply:
Would you have a suggestions here?

Reply:
How about `areTokensForMultipleTenants`? Adding a verb makes it clearer what the function does